openpype/hosts/traypublisher/plugins/create/create_audio.py

```diff
@@ -0,0 +1,51 @@
+import os
+
+from openpype.lib.attribute_definitions import FileDef
+from openpype.pipeline import (
+    CreatedInstance,
+    CreatorError
+)
+from openpype.hosts.traypublisher.api.plugin import TrayPublishCreator
+
+
+class AudioCreator(TrayPublishCreator):
+    """Creates audio instance on asset."""
+
+    identifier = "io.openpype.creators.traypublisher.audio"
+    label = "Audio"
+    family = "audio"
+    description = "Publish audio files."
+    extensions = [".wav"]
```

Comment:
Does it make sense to also add some other popular audio formats like `mp3`? Maybe even `aiff`, `flac`, `aac`, `ogg`?